okabamac/Banka
# ch(travis) refactor codes to fix travis error

## What does this PR do?
Fix Travis build and refactor the get account transactions endpoint code

## Description of Task to be completed?
Get Travis build working
have the following endpoint working
POST/api/v1/accounts/:accountnumber

# How should this be manually tested?
clone the repo git clone git@github.com:okabamac/banka.git

on your terminal run cd journal

run git checkout ch-165593726

run npm install

run npm run dev

make a request to POST localhost:3000/api/v1/auth/users
NB see screenshot below on the request data to pass

Any background context you want to provide?
NA
## What are the relevant pivotal tracker stories?
#165593726
